src/main/java/com/terraformersmc/modmenu/gui/widget/DescriptionListWidget.java

```diff
@@ -272,55 +269,52 @@ public void renderList(DrawContext DrawContext, int mouseX, int mouseY, float de
 		RenderSystem.blendFuncSeparate(GlStateManager.SrcFactor.SRC_ALPHA, GlStateManager.DstFactor.ONE_MINUS_SRC_ALPHA, GlStateManager.SrcFactor.ZERO, GlStateManager.DstFactor.ONE);
 		RenderSystem.setShader(GameRenderer::getPositionColorProgram);
 
-		bufferBuilder.begin(VertexFormat.DrawMode.QUADS, VertexFormats.POSITION_COLOR);
-		bufferBuilder.vertex(this.getX(), (this.getY() + 4), 0.0D).
+		bufferBuilder = tessellator.begin(VertexFormat.DrawMode.QUADS, VertexFormats.POSITION_COLOR);
+		bufferBuilder.vertex(this.getX(), (this.getY() + 4), 0.0F).
 
-				color(0, 0, 0, 0).
+				color(0, 0, 0, 0);
 
-				next();
-		bufferBuilder.vertex(this.getRight(), (this.getY() + 4), 0.0D).
+		bufferBuilder.vertex(this.getRight(), (this.getY() + 4), 0.0F).
 
-				color(0, 0, 0, 0).
+				color(0, 0, 0, 0);
+		
+		bufferBuilder.vertex(this.getRight(), this.getY(), 0.0F).
 
-				next();
-		bufferBuilder.vertex(this.getRight(), this.getY(), 0.0D).
+				color(0, 0, 0, 255);
+		
+		bufferBuilder.vertex(this.getX(), this.getY(), 0.0F).
 
-				color(0, 0, 0, 255).
+				color(0, 0, 0, 255);
+		
+		bufferBuilder.vertex(this.getX(), this.getBottom(), 0.0F).
 
-				next();
-		bufferBuilder.vertex(this.getX(), this.getY(), 0.0D).
+				color(0, 0, 0, 255);
+		
+		bufferBuilder.vertex(this.getRight(), this.getBottom(), 0.0F).
+			
+				color(0, 0, 0, 255);
+		
+		bufferBuilder.vertex(this.getRight(), (this.getBottom() - 4), 0.0F).
 
-				color(0, 0, 0, 255).
-
-				next();
-		bufferBuilder.vertex(this.getX(), this.getBottom(), 0.0D).
-
-				color(0, 0, 0, 255).
-
-				next();
-		bufferBuilder.vertex(this.getRight(), this.getBottom(), 0.0D).
-
-				color(0, 0, 0, 255).
-
-				next();
-		bufferBuilder.vertex(this.getRight(), (this.getBottom() - 4), 0.0D).
-
-				color(0, 0, 0, 0).
-
-				next();
-		bufferBuilder.vertex(this.getX(), (this.getBottom() - 4), 0.0D).
-
-				color(0, 0, 0, 0).
-
-				next();
-		tessellator.draw();
+				color(0, 0, 0, 0);
+		
+		bufferBuilder.vertex(this.getX(), (this.getBottom() - 4), 0.0F).
 
+				color(0, 0, 0, 0);
+		
+		try {
+			builtBuffer = bufferBuilder.end();
+			BufferRenderer.drawWithGlobalProgram(builtBuffer);
+			builtBuffer.close();
+		}
+		catch (Exception ignored) { }
```

Comment:
Nitpick: braces should be on the same lines as keywords.

```suggestion
		try {
			builtBuffer = bufferBuilder.end();
			BufferRenderer.drawWithGlobalProgram(builtBuffer);
			builtBuffer.close();
		} catch (Exception e) {
			// Ignored
		}
```